AdminUi/src/AdminUi.Infrastructure/Persistence/Database/EntityTypeConfigurations/IdentityOverviewEntityTypeConfiguration.cs

```diff
@@ -9,7 +9,14 @@ public class IdentityOverviewEntityTypeConfiguration : IEntityTypeConfiguration<
     public void Configure(EntityTypeBuilder<IdentityOverview> builder)
     {
         builder.ToView("IdentityOverviews");
-        builder.HasNoKey();
+        builder.OwnsOne(
+            c => c.Tier,
+            dt =>
+            {
+                dt.Property(p => p.Id).HasColumnName("TierId");
+                dt.Property(p => p.Name).HasColumnName("TierName");
+            });
+        builder.HasKey(c => c.Address);
```

Comment:
I don't know whether it is a good idea to define a key here. Afaik a view can't have a PK.

Did you run your code on SQL Server as well as Postgres?